docker.go

```diff
@@ -53,6 +53,19 @@ func runDockerCommand(command string, args ...string) (string, string, int) {
 	return runDockerCommandWithTimeout(time.Duration(Timeout), command, args...)
 }
 
+// LogsDockerContainer returns the container logs
+func LogsDockerContainer(name string) (string, error) {
+	args := []string{name}
+
+	stdout, _, exitCode := runDockerCommand("logs", args...)
+
+	if exitCode != 0 || stdout == "" {
```

Comment:
This function fails if not not have stdout, it is possible that check for a container logs should not have any log ? 
Also this function is only used once and the error is ignored.  can you better not fail on non stdout and properly handle the error where is called?

Reply:
Yes you're right, I should not consider an empty stdout as an error.